src/lib/menu/menu-trigger.ts

```diff
@@ -70,6 +70,7 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
     if (!this._menuOpen) {
       this._createOverlay();
       this._overlayRef.attach(this._portal);
+      this._overlayRef.backdropClick().subscribe(() => this.closeMenu());
```

Comment:
Shouldn't it be 

```
this._overlayRef.backdropClick().take(1).subscribe(() => this.closeMenu());
```

to unsubscribe the observer since the backdrop element will be removed from DOM? see https://jsfiddle.net/chy8gLjs/ vs. https://jsfiddle.net/j4p7gq1v/